openslides/core/signals.py

```diff
@@ -106,27 +107,8 @@ def setup_general_config(sender, **kwargs):
             help_text=ugettext_lazy('Use web color names like "red" or hex numbers like "#ff0000".'),
             required=True))
 
-    welcome_title = ConfigVariable(
-        name='welcome_title',
-        default_value=_('Welcome to OpenSlides'),
-        translatable=True,
-        form_field=forms.CharField(
-            widget=forms.TextInput(),
-            label=ugettext_lazy('Title'),
-            help_text=ugettext_lazy('Also used for the default welcome slide.'),
-            required=False))
-
-    welcome_text = ConfigVariable(
```

Comment:
Why welcome_title and welcome_text is removed? We use the title also for the welcome slide.